test/e2e_node/util.go

```diff
@@ -444,7 +439,7 @@ func restartKubelet(running bool) {
 }
 
 // stopKubelet will kill the running kubelet, and returns a func that will restart the process again
-func stopKubelet() func() {
+func stopKubelet(ctx context.Context, f *framework.Framework) func() {
```

Comment:
better name after this change will be:

```suggestion
func mustStopKubelet(ctx context.Context, f *framework.Framework) func() {
```

Reply:
done